scripts/cfgCommons.py

```diff
@@ -49,7 +49,9 @@ def __init__(self, args):
                 for common_configFilePath in args.common_configFilePaths: # go over all the config files and collect all parameters
                     logger.info("Processing config file:" + common_configFilePath)
                     configPart = configparser.ConfigParser()
-                    configPart.read(common_configFilePath)
+                    with openFile(common_configFilePath) as configFile:
+                        configPart.read_file(openFile(common_configFilePath)) 
```

Comment:
I guess that we want to change it to something like: `configPart.read_file(configFile)`

Reply:
Right, fixed in a454042 .